rsv-core/lib/python/rsv/rsv_control.py

```diff
@@ -109,7 +109,7 @@ def process_options(arguments=None):
                      help="Skip the ping test against the host being monitored")
     group.add_option("--ce-type", "--gatekeeper-type", "--gk-type", dest="ce_type", default="gram",
                      help="Which CE type to use for remote tests via Condor-G. "
-                     "Valid values are 'gram' for Globus GRAM, and 'htcondor-ce' (or 'condor-ce') for HTCondor-CE")
+                     "Valid values are 'gram' for Globus GRAM, 'htcondor-ce' (or 'condor-ce') for HTCondor-CE ,cream for CREAM and Nordugrid for nordugrid")
```

Comment:
Should be `"Valid values are 'gram' for Globus GRAM, 'htcondor-ce' (or 'condor-ce') for HTCondor-CE, 'cream' for CREAM-CE and 'nordugrid' for Nordugrid"`